Source/components/Document.js

```diff
@@ -62,7 +62,7 @@ export class Document extends WrappedObject {
    * @return {Layer} A layer object, if one was found.
    */
   getLayerWithID(layerId) {
-    const layer = this._object.documentData().layerWithID_(layerId)
+    const layer = this._object.documentData().layerWithID(layerId)
```

Comment:
This block of code is repeated verbatim on lines 83:87… maybe an opportunity to refactor it into a method?

Reply:
There is a comment from Sam explaining why :)

Reply:
I missed that, it now makes sense. Thanks! : )